cmd/protoc-gen-cpp-tableau-loader/embed/logger.pc.cc

```diff
@@ -53,14 +54,12 @@ void SetDefaultLogger(Logger* logger) {
   g_default_logger = logger;
 }
 
-int Logger::Init(const std::string& path, Level level) {
-  std::string dir = util::GetDir(path);
-  if (!dir.empty()) {
-    // prepare the specified directory
-    int status = util::Mkdir(dir);
-    if (status == -1) {
-      return status;
-    }
+int Logger::Init(const std::filesystem::path& path, Level level) {
+  std::error_code ec;
+  std::filesystem::create_directories(path.parent_path(), ec);
+  if (ec) {
+    std::cerr << "system error: " << ec.message() << std::endl;
```

Comment:
Error message should be stored at `g_err_msg` but not just printing to std::err.